go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/prometheus/client_golang v1.11.0
 	github.com/spiffe/go-spiffe v1.1.0
 	github.com/stretchr/testify v1.7.0 // indirect
+	gitlab.com/f5/nginx/controller/poc/f5-hugo v0.11.5 // indirect
```

Comment:
why is this needed now? shouldn't it be only in `docs/go.mod`?

Reply:
@jputrino

Reply:
Closing this PR, creating another one to make changes only to Docs folder.